programs/lib/flexpret_exceptions.c

```diff
@@ -4,16 +4,27 @@
 #include "flexpret_assert.h"
 
 #include <flexpret.h>
+#include <errno.h>
 
 typedef void (*isr_t)(void);
 
 static isr_t ext_int_handler;
 static isr_t ie_int_handler;
 static isr_t ee_int_handler;
 
+struct thread_ctx_t contexts[NUM_THREADS];
```

Comment:
This is where the registers mentioned in `programs/lib/ctx_switch.S` are stored.